quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -1843,6 +1845,56 @@ public void testConcurrencyConflictUpdateTableDuringFinalTransaction() {
         .hasMessageContaining("conflict_table");
   }
 
+  @ParameterizedTest
+  @ValueSource(booleans = {false, true})
+  public void testTableOperationsDoesNotRefreshAfterCommit(boolean updateMetadataOnCommit) {
+    Assumptions.assumeTrue(
+        requiresNamespaceCreate(),
```

Comment:
Did you mean `assertTrue`? If it's not `true` under an "assumption" the test will simply be skipped, but I guess we'll want to revisit and fix the test if it ever happens :thinking:

Reply:
`Assumptions.assumeTrue` is proliferated all through the existing suite, so if we want to update this we could also do them all at that time

Reply:
fair enough